tests/lib/snaps/apply-config-hook-set/meta/hooks/apply-config

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+echo "Setting a configuration value"
+if ! snapctl set baz=qux; then
```

Comment:
Can we try to accommodate things so we remain with a single snap?  Should be easy to tune behavior based on configuration (!), rather than having N snaps for N possible permutations.